### PR TITLE
ba-147 make scroll bar looking more material

### PR DESCRIPTION
**Pull request by patrick-hebner**

### CODE REVIEW

**Comment by flashback2k14** [comment]
lgtm

File: app/styles/main.css [hunk 1]
```diff
@@ -11,17 +11,19 @@ body {
...
 ::-webkit-scrollbar-thumb {
-  -webkit-border-radius: 10px;
-  border-radius: 10px;
-  background: transparent;
-  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.5);
+  /*-webkit-border-radius: 10px;*/
+  /*border-radius: 10px;*/
+  /*background: transparent;*/
+  /*-webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.5);*/
+  background-color: rgba(49,27,146,1.0);
```
Comment (lyio) [inline app/styles/main.css:18]: Aren't the other colors in hex?

File: app/styles/main.css [hunk 2]
```diff
@@ -11,17 +11,19 @@ body {
 
 /* Track */
 ::-webkit-scrollbar-track {
-  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
-  -webkit-border-radius: 10px;
-  border-radius: 10px;
+  /*-webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);*/
```
Comment (lyio) [inline app/styles/main.css]: Can those lines be removed?

Comment (flashback2k14) [inline app/styles/main.css]: I think this could stay at the moment if want to change this maybe later.

File: app/styles/main.css (hunk 1 above)
Comment (flashback2k14) [inline app/styles/main.css:18]: we have both. hex and rgba.

**Comment by lyio** [comment]
lgtm

**Pull request merged**